Buildings/Experimental/DHC/Networks/Steam/Examples/ConnectionCondensatePipe.mo

```diff
@@ -0,0 +1,106 @@
+within Buildings.Experimental.DHC.Networks.Steam.Examples;
+model ConnectionCondensatePipe "Example model for the steam heating connection block"
+  extends Modelica.Icons.Example;
+
+  package MediumSte = Buildings.Media.Steam "Steam vapor medium";
```

Comment:
In most steam models, you have used "Steam medium" and "Water medium".
`vapor` in medium description can be removed if this was not intentionally written.